src/utils/decorators/input/input.scss

```diff
@@ -195,7 +195,7 @@
   height: 15px;
   position: absolute;
   right: 5px;
-  top: 6px;
+  top: 28px;
```

Comment:
It would be good to release something that solves the problem in a way that doesn't come back like this though, as it looks like it was changed ten days ago to fix some other problem (which might well have come back in the mean time)

However, I can approve this for now, just food for thought